cmd/lncli/commands.go

```diff
@@ -525,7 +532,31 @@ func sendPaymentCommand(ctx *cli.Context) error {
 var ShowRoutingTableCommand = cli.Command{
 	Name:        "showroutingtable",
 	Description: "shows routing table for a node",
-	Action:      showRoutingTable,
+	Usage:       "showroutingtable [--table]",
+	Flags: []cli.Flag{
+		cli.BoolFlag{
+			Name:  "table",
+			Usage: "Show the routing table in table format. Print only a few first symbols of id",
```

Comment:
I think this can be re-worded a bit to make the description a bit clearer: 
> Show the routing table in a table format. Print only the first few symbols of each node's Lightning ID.